frontend/src/components/Content.js

```diff
@@ -1,29 +1,35 @@
 import { Container, Typography } from "@mui/material";
 import React from "react";
 import { UserInfoContext } from "../stores/UserInfoStore";
+import StudentDashBoard from "./StudentDashboard";
 
 const Content = () => {
-  const [userInfo ] = React.useContext(UserInfoContext);
+  const [userInfo] = React.useContext(UserInfoContext);
   return (
-    <Container
-      sx={{
-        display: "flex",
-        justifyContent: "center",
-        alignItems: "center",
-        mt: "10%",
-        textAlign: "center",
-      }}
-    >
-      {!userInfo.loggedIn ?
-        <Typography variant="h3">
-          Bienvenue visiteur.
-        </Typography>
-      :
-      <Typography variant="h3">
-      Bienvenue {userInfo.email.toLowerCase()}
-    </Typography>
-      }
-    </Container>
+    <>
+      <Container
+        sx={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          mt: "10%",
+          textAlign: "center",
+        }}
+      >
+        {!userInfo.loggedIn ? (
+          <Typography variant="h3">Bienvenue visiteur.</Typography>
+        ) : (
+          <Typography variant="h3">
+            Bienvenue {userInfo.email.toLowerCase()}
+          </Typography>
+        )}
+      </Container>
+      {userInfo.role == "STUDENT" ? (
+        <div>
+          <StudentDashBoard></StudentDashBoard>
+        </div>
+      ) : null}
```

Comment:
Is this going to change to a switch when more roles are going to need dashboards or student is the only one with a dash board ?

Reply:
broo c'est juste pour afficher william va override, pcq ca s'affichait tout le temps et je voulait l'afficher pour quand c'est un étudiant et logiquement il y aura probablement d'autre dashboard